CinchKit/Models/PollModel.swift

```diff
@@ -140,14 +140,16 @@ public struct CNHNotification {
     public let href : NSURL
     public let created : NSDate
     public let action : String
-    
-    public let accountFrom : CNHAccount?
-    public let accountTo : CNHAccount?
-    
-    public let resourceId : String
-    public let resourceType : String
-    
-    public let poll : CNHPoll?
+
+    public let senderAccount : CNHAccount?
+    public let senderCategory: CNHCategory?
```

Comment:
1. receiver is needed for counting polls and votes in the category.
2. sender is needed when someone creates a poll in the category. it looks like category tells you that poll was created in the category. mainly i need this to know which polls are in the category and doing different actions with these polls. for example when category for poll was changed